### PR TITLE
Update Delay Mastercopy

### PR DESCRIPTION
**Pull request by cristovaoth**
Screenshot 2023-09-26 at 16 56 40

### CODE REVIEW

File: sdk/contracts.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ const CanonicalAddresses: Record<
     "1.0.0": "0x03B5eBD2CB2e3339E93774A1Eb7c8634B8C393A9",
   },
   [KnownContracts.DELAY]: {
-    "1.0.0": "0xD62129BF40CD1694b3d9D9847367783a1A4d5cB4",
```
Comment (auryn-macmillan) [inline sdk/contracts.ts:86]: Should it be a new version number?

Comment (auryn-macmillan) [reply sdk/contracts.ts:86]: I went ahead and published the latest version as v1.0.1

Comment (cristovaoth) [reply sdk/contracts.ts:86]: Sounds good 👌

Comment (cristovaoth) [reply sdk/contracts.ts:86]: But not on this PR tho, right?

Comment (auryn-macmillan) [reply sdk/contracts.ts:86]: I think we should leave the previous version as 1.0.0 and follow the release versioning on each of the module repos from here on out.

Comment (cristovaoth) [reply sdk/contracts.ts:86]: Right, I for the other modules I remember we use latest, and we only store one version of the ABI. I think accommodating this, is a bigger PR

**Review by auryn-macmillan — APPROVED** [review]
lgtm

**Pull request merged**